ros_buildfarm/templates/release/rpm/mock_config.cfg.em

```diff
@@ -44,6 +44,16 @@ config_opts['macros']['%_without_weak_deps'] = '1'
 config_opts['chroot_setup_cmd'] += ' gcc-c++ make'
 @[end if]@
 
+@[if os_name == 'rhel' and os_code_name == '8']@
+# Work around a broken binutils release in RHEL 8.4
+config_opts['chroot_setup_cmd'] += ' https://ftp.osuosl.org/pub/ros/download.ros.org/downloads/binutils/binutils-2.30-79.el8.x86_64.rpm'
```

Comment:
I see the `chroot_setup_cmd` key getting written to at several points in this template but I don't see it being used. Is the mock.cfg file a Python script itself?

Reply:
Yes, it is!

The `chroot_set_cmd` is essentially a `dnf install -y` command, so appending to it like this is just extending the list of packages getting installed as part of the chroot. `binutils` is normally installed as a dependency during the default chroot initialization, so calling it out as a specific package for installation doesn't change the set of packages being installed.